index.html

```diff
@@ -0,0 +1,863 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Lucent UI</title>
+    <!-- CSS Links -->
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="wrapper">
+        <!-- Header Starts -->
+        <header class="header">
+            <div class="navbar-container">
+                <a class="navbar navbar-m-lr-1" href="#">
+                    <h1 class="navbar-m-0">LUCENT UI</h1>
+                </a>
+                <a class="navbar-m-left-auto navbar" href="#">
+                    <h5>HOME</h5>
```

Comment:
If you are using h5 before that you should also have h2 h3 h4, dont use h5 tags just for sizing of the font.
You can use h2 here and for font size you can use utilty classes